client/docs/cosmos-legacy-swagger.yaml

```diff
@@ -0,0 +1,2598 @@
+---
```

Comment:
Do we need legacy file?

Reply:
There are a few tendermint RPC endpoints, for which we don't have gRPC alertnatives. This legacy swagger has those endpoints. May be we can remove some APIs from there, will update

Reply:
Got it.